source/fundamentals.txt

```diff
@@ -25,20 +25,20 @@ Fundamentals
    /fundamentals/csfle
    /fundamentals/time-series
 
-The Fundamentals section contains detailed information to help you
-expand your knowledge on how to use MongoDB with the Java driver. We
-divide the information in this section into the following topics: 
+The Fundamentals section is where you will find coverage of
+core operations. This section contains information on how to:
 
 - :doc:`Connect to MongoDB </fundamentals/connection>`
-- :doc:`Specify an API Version </fundamentals/versioned-api>`
+- :doc:`Use the Versioned API </fundamentals/versioned-api>`
 - :doc:`Authenticate with MongoDB </fundamentals/auth>`
-- :doc:`Use the Driver's Data Formats </fundamentals/data-formats>`
-- :doc:`Perform CRUD Operations </fundamentals/crud>`
+- :doc:`Convert between MongoDB Data Formats and Java Objects </fundamentals/data-formats>`
+- :doc:`Read from and Write to MongoDB </fundamentals/crud>`
 - :doc:`Simplify your Code with Builders </fundamentals/builders>`
-- :doc:`Perform Aggregations </fundamentals/aggregation>`
-- :doc:`Construct Indexes </fundamentals/indexes>`
-- :doc:`Specify Collations </fundamentals/collations>`
-- :doc:`Record Events in the Driver </fundamentals/logging>`
-- :doc:`Use Driver Events in your Code </fundamentals/monitoring>`
-- :doc:`Store and Retrieve Files in MongoDB </fundamentals/gridfs>`
+- :doc:`Transform your Data </fundamentals/aggregation>`
+- :doc:`Create Indexes to Efficiently Execute Queries </fundamentals/indexes>`
```

Comment:
Nitpick: I think this could be easier for a wider audience to understand:

```suggestion
- :doc:`Create Indexes to Speed Up Queries </fundamentals/indexes>`
```